components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2800,6 +2794,12 @@ components:
           type: integer
           format: int64
           example: 3600
+        bindingType:
+          type: string
+          description: Access token binding type.
```

Comment:
Can we improve the description to say what access token binding is actually?
Basically to give the API consumer an idea as to why he should use this and what he can do with this?

Reply:
Updated with https://github.com/wso2/identity-api-server/pull/186/commits/9a85e7ed286f2a7422abc510521aaa02ccf0fd4a